KaoriMiyazono.json

```diff
@@ -0,0 +1,11 @@
+{
+	"type" : 0,
+	"english-name" : "Miyazono Kaori ",
+	"image" : "https://vignette.wikia.nocookie.net/shigatsu-wa-kimi-no-uso/images/3/3d/Ep03.jpg/revision/latest/scale-to-width-down/300?cb=20150306235014",
```

Comment:
remove the ``/revision/latest/scale-to-width-down/300?cb=20150306235014`` for a larger image